### ISSUE TITLE
Postgres CDC and CloudSQL IAM Service Account throws Null Pointer exception

### ISSUE THREAD

**Issue by benen** [airbytehq/airbyte#22298]
## Environment
- **Airbyte version**:  `0.40.27`
- **OS Version / Instance**: `debian-10-buster-v20221206`
- **Deployment**: Docker compose running on GCE VM. 
- **Source Connector and version**: `airbyte/source-postgres:1.0.36`
- **Destination Connector and version**: `airbyte/destination-bigquery:1.2.9`
- **Step where error happened**: Sync job

## Current Behavior

Airbyte throws a null pointer exception when using a CloudSQL Postgres user for IAM Service Account, only when configured for CDC. 

## Expected Behavior

Sync should complete without exception as per full refresh with the same IAM configuration  The exact same config works fine when using a built in postgres user with an explicit password set. 

## Logs

The relevant exception is

```
2023-01-17 06:50:18 [44msource[0m > Exception occurred while getting the delegate read iterator, closing SSH `tunnel`
Stack Trace: java.lang.NullPointerException
	at java.base/java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1011)
	at java.base/java.util.concurrent.ConcurrentHashMap.put(ConcurrentHashMap.java:1006)
	at java.base/java.util.Properties.put(Properties.java:1301)
	at java.base/java.util.Properties.setProperty(Properties.java:229)
	at io.airbyte.integrations.debezium.internals.PostgresDebeziumStateUtil.connection(PostgresDebeziumStateUtil.java:93)
	at io.airbyte.integrations.debezium.internals.PostgresDebeziumStateUtil.commitLSNToPostgresDatabase(PostgresDebeziumStateUtil.java:114)
	at io.airbyte.integrations.source.postgres.PostgresSource.getIncrementalIterators(PostgresSource.java:368)
	at io.airbyte.integrations.source.postgres.PostgresSource.getIncrementalIterators(PostgresSource.java:79)
	at io.airbyte.integrations.source.relationaldb.AbstractDbSource.read(AbstractDbSource.java:165)
	at io.airbyte.integrations.source.postgres.PostgresSource.read(PostgresSource.java:332)
	at io.airbyte.integrations.base.ssh.SshWrappedSource.read(SshWrappedSource.java:64)
	at io.airbyte.integrations.base.IntegrationRunner.runInternal(IntegrationRunner.java:138)
	at io.airbyte.integrations.base.IntegrationRunner.run(IntegrationRunner.java:98)
	at io.airbyte.integrations.base.adaptive.AdaptiveSourceRunner$Runner.run(AdaptiveSourceRunner.java:86)
	at io.airbyte.integrations.source.postgres.PostgresSourceRunner.main(PostgresSourceRunner.java:15)
```

## Steps to Reproduce

1. Create Postgres Source configured for CDC using an IAM Service Account leaving the password field blank
2. Create a connection
3. Run the sync

## Are you willing to submit a PR?

Potentially. I believe the source of the error due to the code attempting to set a null value in the properties assembled by the Debezium wrapper. Should be a null check methinks before calling the put method.

**Comment by subodh1810** [comment]
Raised a PR https://github.com/airbytehq/airbyte/pull/24271 to fix this

**Comment by subodh1810** [comment]
Fixed in version 2.0.6 postgres
